Block5/Block5_7/HomeWork_Lesson7/ReportService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using TemplateEngine.Docx;
+
+namespace HomeWork_Lesson7
+{
+    public sealed class ReportService
+    {
+        public void GenerateReport(FileSystemInfo fileSystemInfo, string output ="")
+        {
+            if (fileSystemInfo is null)
+            {
+                return;
+            }
+
+            if (string.IsNullOrWhiteSpace(output))
+            {
+                output = Path.Combine(Directory.GetCurrentDirectory(), $"Disk{fileSystemInfo.DiskIndex.ToString()}_FileSystemReport.docx");
+            }
```

Comment:
```suggestion
        public void GenerateReport(FileSystemInfo fileSystemInfo, string output = null)
        {
            if (string.IsNullOrWhiteSpace(output))
            {
                output = Path.Combine(Directory.GetCurrentDirectory(), $"Disk{fileSystemInfo.DiskIndex.ToString()}_FileSystemReport.docx");
            }
```

бессмысленно проверять практически одно и тоже